lib/mutant/mutator/node/send.rb

```diff
@@ -57,6 +57,8 @@ class Send < self
       private
 
         def dispatch
+          return if meta.sorbet_receiver?
```

Comment:
@vaporyhumo This change was partial as it would not prevent mutatn to remove the method call entirely from a "mutation higher up".

the newer system in #1341 fixes this more generic.